### PR TITLE
Minor enhancement to ocean mixing example

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

**Comment by tomchor** [comment]
I can't preview this PR, but I'm guessing this gets rid of the kink in the Δz plot?

File: src/BuoyancyModels/BuoyancyModels.jl [hunk 1]
```diff
@@ -52,10 +52,10 @@ include("g_dot_b.jl")
 include("buoyancy_field.jl")
 
 Base.show(io::IO, b::SeawaterBuoyancy{FT}) where FT =
-    println(io, "SeawaterBuoyancy{$FT}: g = $(b.gravitational_acceleration)", '\n',
+    print(io, "SeawaterBuoyancy{$FT}: g = $(b.gravitational_acceleration)", '\n',
                 "└── equation of state: $(b.equation_of_state)")
 
 Base.show(io::IO, eos::LinearEquationOfState{FT}) where FT =
-    println(io, "LinearEquationOfState{$FT}: ", @sprintf("α = %.2e, β = %.2e", eos.α, eos.β))
+    print(io, "LinearEquationOfState{$FT}: ", @sprintf("α = %.2e, β = %.2e", eos.α, eos.β))
```
Comment (tomchor) [inline src/BuoyancyModels/BuoyancyModels.jl:59]: Can you clarify what this change does?

Comment (glwagner) [reply src/BuoyancyModels/BuoyancyModels.jl:59]: `println` adds a newline after printing the string, I think.

**Review by glwagner — APPROVED** [review]
Seems probable this is an improvement.

File: src/BuoyancyModels/BuoyancyModels.jl (hunk 1 above)
Comment (navidcy) [reply src/BuoyancyModels/BuoyancyModels.jl:59]: yes, that's it... and it was creating this extra lines -->

Screen Shot 2021-11-13 at 11 21 31 am

**Comment by navidcy** [comment]
> I can't preview this PR, but I'm guessing this gets rid of the kink in the Δz plot?

Previews are created at the second commit. But I can paste the figure here. That's the  only difference:

**Pull request merged**